### ISSUE TITLE
Highlight project "titles" in c# build

### ISSUE THREAD

**Issue by pmiossec** [nbalakin/VSOutputEnhancer#21]
Is it possible to highlight, in a c# solution (maybe also applicable for other solution), to highlight the lines corresponding to the project "title" lines?

That's the lines begining and ending by '------' , that you could see on this screenshot:

![image](https://user-images.githubusercontent.com/460196/50662384-75fccb80-0fa6-11e9-886e-19a7fdfcb800.png)

That way it would have been **a lot** easier in the "Build older" output view to spot in which project an error or a warning is displayed. 

I let you choose the color ;) Perhaps cyan, like the color used by msbuild in the console...

PS: Thanks a lot for this very useful extension!

**Comment by pmiossec** [comment]
>Why do you need such highlight? I mean, what is the information you want to get from the log by highlighting "build started" lines?

The project name. When I see a warning/error, I would like to know to which project it belongs.

**Comment by nbalakin** [comment]
I'm not sure it will be useful in such scenarios because MSBuild in parallel mode building multiple projects at the same time so you will need to find a header with the same runner id. Here's an example:

```
1>------ Rebuild All started: Project: FirstDependency, Configuration: Debug Any CPU ------
2>------ Rebuild All started: Project: SecondDependency, Configuration: Debug Any CPU ------
1>  SomeFile.cs: warning example: A warning from FirstDependency.csproj
1>  FirstDependency -> ...\FirstDependency\bin\Debug\FirstDependency.dll
2>  SomeFile.cs: warning example: A warning from SecondDependency.csproj
2>  SecondDependency -> ...\SecondDependency\bin\Debug\SecondDependency.dll
3>------ Rebuild All started: Project: ConsoleApp, Configuration: Debug Any CPU ------
3>  ConsoleApp -> ...\ConsoleApp\bin\Debug\ConsoleApp.exe
========== Rebuild All: 3 succeeded, 0 failed, 0 skipped ==========
```

**Comment by pmiossec** [comment]
I admit that you get the best value of this feature when using the "Build order" output.

But even when using the "Build" output, it could be helpful when using the project number `1>` to find which project displayed the warning or errors.

PS: You could see what it look likes in the PR I made #22

**Comment by nbalakin** [comment]
Just status update so you don't think the issue is forgotten. 🙂 

I'm trying to do something with the extension color scheme and add there a new color that is not too bright but still noticeable if you're looking for it.

I'd offer you to split PR and merge all the parsing stuff and leave the JSONs as a separate PR. Meanwhile I'll implement #2 so everybody will be happy.

**Comment by pmiossec** [comment]
Any update on this (and #22)? I would like all my team to benefit from the PR...
